integr8ly/cloud-resource-operator
# [INTLY-7222] Create a skeleton controller for metrics reconciliation in CRO

## Overview

Work in progress 
Jira: https://issues.redhat.com/browse/INTLY-7222

## Verification

- Clone this branch
- Run `make cluster/prepare`
- Run `make run`
- Ensure that the new controller is registered in the operator logs 
- Ensure that the controller is being reconciled every 5 minutes
